quic/transport/framesorter.nim

```diff
@@ -0,0 +1,78 @@
+import ../errors
+import std/[tables, strformat]
+import chronos
+import std/[algorithm, sequtils]
+
+type
+  Gap = object
+    start, finish: uint64 # end_ is exclusive
```

Comment:
nit: `finish` -> `end` other part of same file uses `end` (more precisely `endPos`).

Reply:
renamed it to `startPos` and `endPos` since `end` is a reserved word